packages/psnotify.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,13 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+$toolName = 'psnotify'
+$category = 'dotNet'
+
+$executablePath = 'C:\psnotify'
+Move-Item -Path $executablePath -Destination ${Env:RAW_TOOLS_DIR}
```

Comment:
I think it would be better to add the logic of `VM-Uninstall` here instead of moving the directory just to delete it with the helper. I find it a bit confusing.